### ISSUE TITLE
Add swipe to refresh lists

### ISSUE THREAD

**Issue by mboutell** [AndroidCourseMaterial/GradeRecorderFirebase#6]
Swiping up on a recyclerview will refresh the data from Firebase.
See http://developer.android.com/training/swipe/add-swipe-interface.html and http://developer.android.com/reference/android/support/v4/widget/SwipeRefreshLayout.html

**Comment by rockwotj** [comment]
Shouldn't the data automatically refresh from Firebase with the listeners?

**Comment by mboutell** [comment]
Good point. I'm still in Appengine-land. The swipe feature just seemed so nice - can you think of any place we could use it?

**Comment by rockwotj** [comment]
It is very nice to use, and I can't think of anywhere that you need to refresh with firebase, unless you purposefully use a single event listener, it might be a bit more efficient if you're sure the data won't change frequently. But that's kind of forcing the use of the refresh.
